package.json

```diff
@@ -41,9 +41,10 @@
   "dependencies": {
     "fs-extra": "^7.0.0",
     "probot": "^7.4.0",
+    "request": "^2.88.0",
     "tslint": "^5.14.0",
-    "typescript": "^3.3.4000",
-    "tslint-config-security": "^1.15.0"
+    "tslint-config-security": "^1.15.0",
```

Comment:
Is there any reason to reorder these two lines?

Reply:
It was automatically generated that way :D